### ISSUE TITLE
why got worse result after finetune?

### ISSUE THREAD

**Issue by andysun-sw** [facebookresearch/ParlAI#4353]
i use following ways to finetune reddit_3B on blender skill task(just as https://parl.ai/projects/recipes/):

parlai train_model -t blended_skill_talk  -veps 2.5 --attention-dropout 0.0 --batchsize 16 --model transformer/generator --embedding-size 2560 --ffn-size 10240 --variant prelayernorm --n-heads 32 --n-positions 128 --n-encoder-layers 2 --n-decoder-layers 24 --history-add-global-end-token end --delimiter '  ' --dict-tokenizer bytelevelbpe  --dropout 0.1 --fp16 True --init-model zoo:blender/reddit_3B/model --dict-file zoo:blender/reddit_3B/model.dict --label-truncate 128 --log_every_n_secs 10 -lr 7e-06 --lr-scheduler reduceonplateau --lr-scheduler-patience 3 --optimizer adam --relu-dropout 0.0 --activation gelu --model-parallel true --save-after-valid True --text-truncate 128 --truncate 128 --warmup_updates 100 --fp16-impl mem_efficient --update-freq 2 --gradient-clip 0.1 --skip-generation True -vp 4 -vmt ppl -vmm min --model-file /py_torch/parlai_finetune_model/test_train_27B

**but when I display_model, I can't get any result, all of the response are "model:no response"**

![Image](https://user-images.githubusercontent.com/89009298/153703386-c5ae51e7-4eb2-4abb-ada4-a116f9952b2c.png)
if using parlai interactive, system can response,but just simple sentences.
![Image](https://user-images.githubusercontent.com/89009298/153703584-1c7b5141-1948-4ff8-bbff-f7bb7f8c89c2.png)

but when i use 
parlai display_model  -t blended_skill_talk -mf zoo:blender/reddit_3B/model   
the system will response good result, though some repeation. That means the finetune made response worse
![Image2](https://user-images.githubusercontent.com/89009298/153703596-2feab58a-d87c-4b61-bff0-44fb96bf263c.png)

**Comment by klshuster** [comment]
`--skip-generation` defaults to the value used during training, `True`. you'll need to set that to `False`. You can also set `--init-opt gen/blenderbot` to automatically set all of the generation settings to be the same as the original blenderbot model.

**Comment by klshuster** [comment]
Can you try pulling from master?

**Comment by jh-skaion** [comment]
how can i do that? 
thank you

**Comment by klshuster** [comment]
how have you installed parlai on your machine?

**Comment by jh-skaion** [comment]
i used the "pip install parlai" command

**Comment by klshuster** [comment]
which parlai version are you running?

alternatively, you can look at the options corresponding to `gen/blenderbot` at this page, and just specify those manually https://parl.ai/docs/opt_presets.html
